pkg/services/cleanup/cleanup.go

```diff
@@ -151,3 +152,14 @@ func (srv *CleanUpService) expireOldUserInvites() {
 		srv.log.Debug("Expired user invites", "rows affected", cmd.NumExpired)
 	}
 }
+
+func (srv *CleanUpService) deleteStaleShortURLs() {
+	cmd := models.DeleteShortUrlCommand{
```

Comment:
I think we want to stop using the bus. You can either inject ShortURLService like done here https://github.com/grafana/grafana/blob/b91e9faccf1a6a2a97f844bf25d4f914c6f3a8f6/pkg/api/http_server.go#L77 or either do something like this https://github.com/grafana/grafana/blob/b91e9faccf1a6a2a97f844bf25d4f914c6f3a8f6/pkg/services/auth/token_cleanup.go#L10

Reply:
Forgot to push a commit 😅  Yeah I noticed that pattern before when doing the cleanup for invites. Will check next week if it's worth sticking with this